model/tab/gear/gear-main.html

```diff
@@ -3,17 +3,27 @@
         <label>{{localize "GEAR.WEIGHT"}}</label>
         <select name="data.weight">
             {{#select data.weight}}
-                <option value="none">{{localize "WEIGHT.NONE"}}</option>
-                <option value="tiny">{{localize "WEIGHT.TINY"}}</option>
-                <option value="light">{{localize "WEIGHT.LIGHT"}}</option>
-                <option value="regular">{{localize "WEIGHT.REGULAR"}}</option>
-                <option value="heavy">{{localize "WEIGHT.HEAVY"}}</option>
+                <optgroup label={{localize "WEIGHT.TYPE_REGULAR" }}>
```

Comment:
Wondering if using `<optgroup>` is the way to go or if I should just do a disabled choice over the special encumbrance. 

There is the argument of not splitting at all, but I feel some separation is needed to avoid confusing newer players.

Reply:
This seems like a good solution to me.